metaflow/plugins/kfp/kfp.py

```diff
@@ -1130,15 +960,22 @@ def create_workflow_uid_op(s3_sensor_path: str):
                     "volume" in s.resource_requirements
                     for s in step_to_kfp_component_map.values()
                 ):
-                    workflow_uid_op = func_to_container_op(
-                        get_workflow_uid,
-                        base_image="gcr.io/cloud-builders/kubectl",
-                    )(
-                        work_flow_name="{{workflow.name}}",
-                        s3_sensor_path=s3_sensor_path,
-                    ).set_display_name(
-                        "get_workflow_uid"
-                    )
+                    get_workflow_uid_command = [
+                        "bash",
+                        "-ec",
+                        self._compile_time_bootstrap_cmd
+                        + (
```

Comment:
style: this can be done with consistent `f""`:

```python
                       
                        (
                            f"{self._compile_time_bootstrap_cmd}"
                            " && python -m metaflow.plugins.kfp.kfp_get_workflow_uid"
                            f" --s3_sensor_path '{s3_sensor_path}'"
                            " --workflow_name {{workflow.name}}"
                        ),